src/meson.build

```diff
@@ -860,7 +855,7 @@ cmake_package_prefix_dir = '${CMAKE_CURRENT_LIST_DIR}/@0@'.format(cmake_package_
 cmake_install_includedir = get_option('includedir')
 
 if fs.is_absolute(cmake_install_includedir)
-  if have_fs_relative_to
+  if meson.version().version_compare('>=1.3.0')
```

Comment:
I don’t like that we now have this version check allover the place. I’d say it is a meson bug that it can’t detect that `have_fs_relative_to` is only true for `meson.version().version_compare('>=1.3.0')`.

Reply:
I'll remove this one if you want.

Reply:
I think it is OK, we will upgrade to >=1.3 one day...